src/parsers/buddyinfo.cpp

```diff
@@ -66,7 +67,15 @@ zone parse_buddyinfo_line(const std::string& line)
 
         zn.name = tokens[ZONE_NAME];
 
-        for (size_t i = 0; i < zn.chunks.size(); ++i)
+        // std::distance can be negative, but the conditional on line 46
```

Comment:
Instead of assuming the line no. would stay the same, can we just say "the tokens size condition above"?